src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -727,13 +714,48 @@ static int enumerate_partitions(dev_t devnum) {
                         r = k;
         }
 
-        if (m->partitions[PARTITION_XBOOTLDR].found) {
+        esp_state = m->partitions[PARTITION_ESP].found ? ESP_NEEDS_CHECK : ESP_DONT_MOUNT;
+        xbootldr_ok = m->partitions[PARTITION_XBOOTLDR].found;
+
+        if (is_efi_boot()) {
+                sd_id128_t loader_uuid;
+
+                r = efi_loader_get_device_part_uuid(&loader_uuid);
+                if (r < 0) {
+                        /* Won't be able to check if this is the boot ESP, thus not mounting it */
+                        esp_state = ESP_DONT_MOUNT;
+                        if (r == -ENOENT)
+                                log_debug("EFI loader partition unknown.");
+                        else
+                                log_error_errno(r, "Failed to read loader partition UUID: %m");
```

Comment:
The error will be ignored now. So, please downgrade the log level. I think, `log_debug_errno()` is sufficient. At most, `log_info_errno()`.
Then, append ", ignoring" in the message.